game_engine/tic_tac_toe_game.py

```diff
@@ -1,23 +1,100 @@
-from typing import Callable
 from copy import deepcopy
+from typing import Callable, List, Optional
+from game_engine import TicTacToeTurn, TicTacToeGameInfo, AbstractTicTacToeGame
```

Comment:
You should separate imports into two groups: one for libraries and one for your own code imports.